dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports.controller/index.test.ts

```diff
@@ -135,24 +140,75 @@ describe('getUtilisationReports', () => {
       },
     };
 
-    const excludeNotUploaded = 'true';
+    const excludeNotReceived = 'true';
 
-    const mockUtilisationReports: UtilisationReport[] = [];
-    jest.mocked(getManyUtilisationReportDetailsByBankId).mockResolvedValue(mockUtilisationReports);
+    const mockUtilisationReports: UtilisationReportEntity[] = [];
+    const findAllByBankIdMock = jest.fn().mockResolvedValue(mockUtilisationReports);
+    jest.spyOn(UtilisationReportRepo, 'findAllByBankId').mockImplementation(findAllByBankIdMock);
 
-    const { req, res } = getHttpMocks({ reportPeriod: getReportPeriodJsonObject(validReportPeriod), excludeNotUploaded });
+    const { req, res } = getHttpMocks({ reportPeriod: getReportPeriodJsonObject(validReportPeriod), excludeNotReceived });
 
     // Act
     await getUtilisationReports(req, res);
 
     // Assert
-    expect(getManyUtilisationReportDetailsByBankId).toHaveBeenCalledWith(bankId, {
+    expect(findAllByBankIdMock).toHaveBeenCalledWith(bankId, {
       reportPeriod: validReportPeriod,
-      excludeNotUploaded: true,
+      excludeNotReceived: true,
     });
 
     expect(res.statusCode).toEqual(200);
     // eslint-disable-next-line no-underscore-dangle
     expect(res._getData()).toEqual(mockUtilisationReports);
   });
+
+  it('maps entities to response', async () => {
+    // Arrange
+    const validReportPeriod: ReportPeriod = {
+      start: {
+        month: 1,
+        year: 2024,
+      },
+      end: {
+        month: 2,
+        year: 2025,
+      },
+    };
+
+    const excludeNotReceived = 'true';
+
+    const mockUtilisationReports = [UtilisationReportEntityMockBuilder.forStatus(UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION).build()];
+    const findAllByBankIdMock = jest.fn().mockResolvedValue(mockUtilisationReports);
+    jest.spyOn(UtilisationReportRepo, 'findAllByBankId').mockImplementation(findAllByBankIdMock);
+
+    const { req, res } = getHttpMocks({ reportPeriod: getReportPeriodJsonObject(validReportPeriod), excludeNotReceived });
+
+    // Act
+    await getUtilisationReports(req, res);
+
+    // Assert
+    expect(findAllByBankIdMock).toHaveBeenCalledWith(bankId, {
+      reportPeriod: validReportPeriod,
+      excludeNotReceived: true,
+    });
+
+    expect(res.statusCode).toEqual(200);
+    // eslint-disable-next-line no-underscore-dangle
+    const responseData = res._getData() as GetUtilisationReportResponse[];
+    expect(responseData.length).toEqual(1);
+    expect(responseData[0].id).toEqual(mockUtilisationReports[0].id);
```

Comment:
ditto to other comment, although I'd imagine it this might just be the best way and is quite simple